jest.config.js

```diff
@@ -23,6 +23,7 @@ const config = {
   modulePaths: ['<rootDir>/src/'],
   moduleNameMapper: {
     '^.+\\.(css|scss)$': '<rootDir>/src/lib/utils/jest/__mocks__/styleMock.js',
+    '^swiper.*$': '<rootDir>/src/lib/utils/jest/__mocks__/moduleMock.js',
```

Comment:
Did you end up needing this?

Reply:
yep, tests fail if swiper module is not mocked :/

Reply:
no worries just wanted to make sure